python/build_images.py

```diff
@@ -0,0 +1,10 @@
+#! /usr/bin/python3.6
+
+import subprocess
+
+string=subprocess.check_output(["find", ".", "-name", "*.drawio", "-print"])
+arr=string.splitlines()
+for s in arr:
+    sD=s.decode("utf-8")
+    subprocess.run(["drawio-batch", sD, 'source/images/'  + sD[22:-7] + ".png"])
```

Comment:
`subprocess.run` needs to have the `check=True` set otherwise it could fail silently.

https://docs.python.org/3.6/library/subprocess.html#subprocess.run